### ISSUE TITLE
Can't start demo with imageflow_server start --demo

### ISSUE THREAD

**Issue by nrkn** [imazen/imageflow#115]
Browser reports:
```
Internal Server Error
Info:"http://raw.githubusercontent.com/imazen/imageflow/04a0d2ab012d44ba9da078cc68358bf076c0e9f8/imageflow_server/demo/index.html"
Error:UpstreamHyperError(Io(Error { repr: Custom(Custom { kind: InvalidInput, error: StringError("Invalid scheme for Http") }) }))
```

No errors in CLI:

```
C:\Users\Nik Coughlin\Dropbox\imageflow>imageflow_server start --demo
Open your browser to http://localhost:39876/proxied_demo/index.html
built 9 hours ago - unofficial CI release build of 0.0.14* (master) for x86_64-pc-windows-msvc sandybridge (2017-01-24)
Listening on localhost:39876
```

**Comment by nathanaeljones** [comment]
Thanks! I think I've been able to reproduce this (github is now forcing an HTTPS redirect).

**Comment by nrkn** [comment]
I know this is not a helpful comment, but the first time I ran it I also got some errors in the CLI, unfortunately I closed the command window without copying them and on subsequent runs I only get the browser error

**Comment by nathanaeljones** [comment]
We currently log certain kinds of 404s - those that don't match a mount path.

**Comment by nathanaeljones** [comment]
This should be fixed in v0.0.15

https://github.com/imazen/imageflow/releases/tag/v0.0.15
